### PR TITLE
Issue 64

### LINKED ISSUE #64: stop dice reroll when roll count or remaining rounds = 0

### PR DESCRIPTION
**Pull request by abullockuno**
closes #64

### CODE REVIEW

**Comment by abullockuno** [comment]
also closes #66 apparently

File: project/yahtzee/frontend/templates/board.html [hunk 1]
```diff
@@ -264,7 +264,9 @@ <h1>Select Dice To Save:</h1>
                 <br>
                 <input type="hidden" id="playerId" value="{{ score.player.id }}" />
                 <input type="hidden" id="gameId" value="{{ game.game_id }}" />
-                <a href="{% url 'end' %}">End Game |</a>
+                {% if game.host == hand.player and hand.rem_rounds < 0 %}
```
Comment (ppeters0502) [inline project/yahtzee/frontend/templates/board.html:267]: Should this be == 0 instead of less than 0?

Comment (abullockuno) [reply project/yahtzee/frontend/templates/board.html:267]: hand.rem_rounds is -1 once the last score is input. 

the same logic is used here:
{% if hand.rem_rounds < 0 %}
                  <h1>Game Over!</h1>
                  <h1>Final Score: {{score.sumValues}}</h1>

Comment (ppeters0502) [reply project/yahtzee/frontend/templates/board.html:267]: Ahhh gotcha, my apologies!

**Pull request merged**